.evergreen/provision-remote

```diff
@@ -39,7 +41,10 @@ do_ssh "$target" '
   else
     sudo apt-get update &&
     sudo env DEBIAN_FRONTEND=noninteractive apt-get -y install rsync psmisc
-  fi
+  fi &&
+  
+  curl -fL --retry 3 https://github.com/p-mongodb/deps/raw/main/ubuntu1804-python37.tar.xz | \
```

Comment:
It would be great if we move these dependencies to some mongodb repo/resource.

Reply:
Yes that would be preferable.